docs/install/binary.mdx

```diff
@@ -154,10 +154,6 @@ Ghostty is also available in [Terra](https://terra.fyralabs.com/).
  dnf install ghostty
  ```
 
-<Warning>
```

Comment:
I believe this was from before the separate "Linux (Community)" section was added, it seemed duplicative of line 136 to me. But happy to put it back (and add a similar warning to the new section below) if desired.

Reply:
I'm keeping them on purpose. I really want to ensure people see this message if they just snap click to the distro of choice in the sidebar.

Reply:
:+1: Makes sense. I amended my commit to keep the warning above and a similar one to my new section.